### PR TITLE
Updated file

### PR DESCRIPTION
**Pull request by lyonza**
Applied code change suggestions

### CODE REVIEW

File: lyonza/WhiteTextBroadcast.py [hunk 1]
```diff
@@ -256,11 +256,11 @@ def _add_series_count_text_no_season(self, titled_image: Path) -> Path:
             f'convert "{titled_image.resolve()}"',
             *self.__series_count_text_global_effects(),
             f'-font "{self.EPISODE_COUNT_FONT.resolve()}"',
-            f'-gravity center',
+            f'-gravity northwest',
             *self.__series_count_text_black_stroke(),
-            f'-annotate -1400-750 "{self.episode_text}"',
+            f'-annotate "{self.episode_text}"',
```
Comment (CollinHeist) [inline lyonza/WhiteTextBroadcast.py]:
```suggestion
            f'-annotate +25+25 "{self.episode_text}"',
```
You need to specify some sort of offset (probably not +25+25) for the text

File: lyonza/WhiteTextBroadcast.py [hunk 2]
```diff
@@ -256,11 +256,11 @@ def _add_series_count_text_no_season(self, titled_image: Path) -> Path:
...
             *self.__series_count_text_global_effects(),
             f'-font "{self.EPISODE_COUNT_FONT.resolve()}"',
-            f'-gravity center',
+            f'-gravity northwest',
             *self.__series_count_text_black_stroke(),
-            f'-annotate -1400-750 "{self.episode_text}"',
+            f'-annotate "{self.episode_text}"',
             *self.__series_count_text_effects(),
-            f'-annotate -1400-750 "{self.episode_text}"',
+            f'-annotate "{self.episode_text}"',
```
Comment (CollinHeist) [inline lyonza/WhiteTextBroadcast.py]:
```suggestion
            f'-annotate +25+25 "{self.episode_text}"',
```
You need to specify some sort of offset (probably not +25+25) for the text

**Review by CollinHeist — CHANGES_REQUESTED** [review]
- [x] We need to add annotate positioning (otherwise the card won't work)

File: lyonza/WhiteTextBroadcast.py (hunk 2 above)
Comment (Wdvh) [reply lyonza/WhiteTextBroadcast.py]: I updated my WhiteTextAbsolute.py yesterday you can find the new offset there.

**Review by CollinHeist — APPROVED** [review]

File: lyonza/WhiteTextBroadcast.py [hunk 3]
```diff
@@ -280,10 +275,10 @@ def is_custom_font(font: 'Font') -> bool:
         :returns:   True if a custom font is indicated, False otherwise.
         """
 
-        return ((font.file != WhiteTextBroadcast.TITLE_FONT)
+        return ((font.file != WhiteTextAbsolute.TITLE_FONT)
```
Comment (CollinHeist) [inline lyonza/WhiteTextBroadcast.py]:
```suggestion
        return ((font.file != WhiteTextBroadcast.TITLE_FONT)

```

File: lyonza/WhiteTextBroadcast.py [hunk 4]
```diff
@@ -280,10 +275,10 @@ def is_custom_font(font: 'Font') -> bool:
         :returns:   True if a custom font is indicated, False otherwise.
         """
 
-        return ((font.file != WhiteTextBroadcast.TITLE_FONT)
+        return ((font.file != WhiteTextAbsolute.TITLE_FONT)
             or (font.size != 1.0)
-            or (font.color != WhiteTextBroadcast.TITLE_COLOR)
-            or (font.replacements != WhiteTextBroadcast.FONT_REPLACEMENTS)
+            or (font.color != WhiteTextAbsolute.TITLE_COLOR)
+            or (font.replacements != WhiteTextAbsolute.FONT_REPLACEMENTS)
```
Comment (CollinHeist) [inline lyonza/WhiteTextBroadcast.py]:
```suggestion
            or (font.color != WhiteTextBroadcast.TITLE_COLOR)
            or (font.replacements != WhiteTextBroadcast.FONT_REPLACEMENTS)

```

**Review by CollinHeist — APPROVED** [review]

**Review by CollinHeist — APPROVED** [review]

**Pull request merged**